### ISSUE TITLE
Load balance and deployment

### ISSUE THREAD

**Issue by vnguyenau** [microsoftgraph/microsoft-graph-comms-samples#255]
I made a test with 2 recording bot instances in 2 VMs behind load balancer. I used remote debugger and I saw actions(mute/unmuted/hold etc.) will be sticked to correct instance. I could not find the code did that. When starting the call the request sometimes goes to both vms but after connected everything just in one instance properly.

Also adding more vm is not an issue but automatically scaling down seems to impact to current calls that handle by the instance. How could we make sure load balancer only removes instance if no current call in it. In addition how to deploy new version without outage or lose current calls.

**Comment by stefan2410** [comment]
@vnguyenau I've also the same question about the scaling down and deployment.  
I've also a question to you. You wrote that you can see actions (mute/unmuted/hold etc.). 
I saw them also in the notifications endpoint (change=update). But the event OnCallUpdated waas not raised, neither the Call.Resource.State was changed.
The ParticipantsUpdated or ParticipantUpdated was raised, but without meaningful information. Only that one participant has media streams inactive. How did you solve the problem?

**Comment by vnguyenau** [comment]
Actually I did not get any details about action like mute/unmuted/hold. I only see it made a callback to PlatformCallController\OnNotificationRequestAsync

**Comment by stefan2410** [comment]
@vnguyenau  yes, I've another open issue about the CallState, waiting an answer, thank you.
